src/client.rs

```diff
@@ -47,7 +47,7 @@ use crate::world::{WorldId, Worlds};
 use crate::{ident, LIBRARY_NAMESPACE};
 
 /// Contains the [`ClientEvent`] enum and related data types.
-mod event;
+pub mod event;
```

Comment:
There is a `pub use event::*;` that exports everything so this doesn't need to be public.
```suggestion
mod event;
```